analytics/account_creation.js

```diff
@@ -5,13 +5,14 @@
 // Created account (via email)
 $(document).on(
   'submit',
-  '.auth-register form, .marketing-signup-modal form',
+  '.auth-register form, .marketing-signup-modal form, .artist-page-cta-overlay__register form',
   function() {
-    $(document).one('ajaxComplete', function(e, xhr) {
+    $(document).one('ajaxComplete', function(e, xhr, options) {
```

Comment:
According to @anipetrov we want to track `context` for account creations client-side for the CTA. This passes it along for creation via email.